testplan/docs/director/installation/tc-install-ssd-jiva.md

```diff
@@ -0,0 +1,98 @@
+---
+id: tc-install-ssd-jiva
+title: DOP Install on GCP with Jiva
+sidebar_label: TC-Install-SSD-Jiva
+---
+------
+
+
```

Comment:
Not Supported. (Let's have this in strategy stating only supported methods will be)
 - Standard 
- OpenEBS localPV (Recommended) 
- OpenEBS cStor 
- OpenEBS Jiva (Not supported)